### ISSUE TITLE
ARM builds for EKS

### ISSUE THREAD

**Issue by GEverding** [scylladb/scylla-operator#1216]
* Feature Request
I'd like to be able to deploy to arm instances on EKS. 

**What should the feature do:**

**What is use case behind this feature:**
- Arm deploys on gravitron

**Additional Information:**

**Comment by mflendrich** [comment]
@rzetelskik is this a duplicate of https://github.com/scylladb/scylla-operator/issues/1412 ?
Do we need to do anything else to claim EKS ARM support?

**Comment by mflendrich** [comment]
Assigning to @rzetelskik until this is scoped out / completed.

**Comment by mflendrich** [comment]
Assigning to @rzetelskik until this is scoped out / closed as already done.

**Comment by rzetelskik** [comment]
> Do we need to do anything else to claim EKS ARM support?

We don't test EKS at all in our CI. @grzywin does SCT cover Operator on EKS on arm64?

**Comment by grzywin** [comment]
@mflendrich @rzetelskik Yes, we have some functional tests and longevity tests with EKS and ARM64 (AWS Graviton2)

**Comment by rzetelskik** [comment]
Thanks for confirming @grzywin.
@mflendrich nothing left to do then, closing as completed.
